src/shared/ListItem/ListItem.tsx

```diff
@@ -56,11 +56,11 @@ const ListItem = ({ common, onClickFunc, editFunc, deleteFunc }: PropTypes) => {
 
   return (
     <Swipeable rightButtons={buttons.length > 0 ? buttons : null}>
-      <View style={styles.item}>
-        <TouchableWithoutFeedback onPress={handleOnClick}>
+      <TouchableWithoutFeedback onPress={handleOnClick}>

```

Comment:
When I tried that I had the problem that the "onPress" hook was calld even if I just wanted to slide the item. Is that solved somehow or is it a different behaviour on an other device?

Reply:
on my Android, the onPress is not called, check on your phone please